src/Controllers/UserController.js

```diff
@@ -134,6 +134,27 @@ export class UserController extends AdaptableController {
     });
   }
 
+  resendVerificationEmail(username, token) {
+    var where = {};
+    if (username) {
+      where.username = username;
+    }
+    if (token) {
+      where._email_verify_token = token;
+    }
```

Comment:
we shouldn't match the token, because it's in a scenario where the user got 2 verification emails and the token is different on the 1st one. Thus if they used the 1st email to get a resend, it wouldn't work with this, but it should.